packages/rainbowkit/src/wallets/getInjectedConnector.ts

```diff
@@ -75,36 +88,32 @@ function getInjectedProvider({
   return window.ethereum;
 }
 
-function createInjectedConnector(provider?: any): CreateConnector {
-  return (walletDetails: WalletDetailsParams) => {
-    // Create the injected configuration object conditionally based on the provider.
-    const injectedConfig = provider
-      ? {
-          target: () => ({
-            id: walletDetails.rkDetails.id,
-            name: walletDetails.rkDetails.name,
-            provider,
-          }),
-        }
-      : {};
+function createInjectedConnector({
+  id,
+  name,
+  provider,
+}: CreateInjectedConnectorParameters) {
+  // Create the injected configuration object conditionally based on the provider.
+  const injectedConfig = provider
+    ? {
+        target: () => ({
+          id,
+          name,
+          provider,
```

Comment:
Those return types are required in wagmi. Reference to the docs [here](https://wagmi.sh/core/api/connectors/injected#injected).